app/mailers/application_mailer.rb

```diff
@@ -1,4 +1,11 @@
 class ApplicationMailer < ActionMailer::Base
   default from: "noreply@workdey.com"
   layout "mailer"
-end
+  default to: [
+                "olaide.ojewale@andela.com",
```

Comment:
Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.